### PR TITLE
Fix typo in compiler plugin doc

### PR DESCRIPTION
**Pull request by eygraber**

### CODE REVIEW

**Review by schalkms — APPROVED** [review]

**Comment by schalkms** [comment]
@cortinico context switchting 😉

File: website/docs/gettingstarted/compilerplugin.mdx [hunk 1]
```diff
@@ -28,7 +28,7 @@ plugins {
 
 ## Configuring the Compiler Plugin
 
-The compiler plugin can be configured using the `react{}` block in your gradle file.
```
Comment (cortinico) [inline website/docs/gettingstarted/compilerplugin.mdx:31]: This is me working too much 😅

**Comment by cortinico** [comment]
@eygraber don't you mind updating also the versioned page for 1.23?

**Comment by eygraber** [comment]
@cortinico https://github.com/detekt/detekt/pull/6138

**Pull request merged**